gensim/parsing/preprocessing.py

```diff
@@ -40,6 +40,26 @@
 
 
 def remove_stopwords(s):
```

Comment:
Since we're refactoring, the set of stopwords should be a parameter: `remove_stopwords(s, stopwords=STOPWORDS)`.

IIRC @menshikh-iv had plans to remove this entire package, so not sure if this is relevant.

Reply:
@piskvorky as I investigate, this package is actively used, for this reason, this will be moved (and slightly refactored).